coins/src/adapters/tokenMapping.json

```diff
@@ -6160,6 +6160,11 @@
       "to": "coingecko#ondo-us-dollar-yield",
       "decimals": 18,
       "symbol": "ausdy"
+    },
+    "uusdn": {
+      "to": "coingecko#m-2",
```

Comment:
This API ID wont return price data because its `priced by coingecko`

Reply:
Hmmm, the other tokens from Noble also have Coingecko?

Reply:
we can merge it but i dont think this will add any price support